tests/unit/helpers.js

```diff
@@ -1470,3 +1474,68 @@ export const createExperimentData = ({
 }) => {
   return { [id]: variantId };
 };
+
+const queryByClassName = (container, className) => {
+  return container.querySelectorAll(`.${className}`);
+};
+
+const getByClassName = (container, className) => {
+  const elements = queryByClassName(container, className);
+  if (!elements.length) {
+    return null;
+  }
+  if (elements.length > 1) {
+    throw new Error(`More than one element found for className ${className}`);
+  }
+  return elements[0];
+};
+
+const queryByTagName = (container, tagName) => {
+  return container.getElementsByTagName(tagName);
+};
+
+const getByTagName = (container, tagName) => {
+  const elements = queryByTagName(container, tagName);
+  if (!elements.length) {
+    return null;
+  }
+  if (elements.length > 1) {
+    throw new Error(`More than one element found for tag ${tagName}`);
+  }
+  return elements[0];
+};
+
+export const render = (ui, options = {}) => {
+  const internalOptions = {
+    history: addQueryParamsToHistory({ history: createMemoryHistory() }),
+    i18n: fakeI18n(),
+    store: dispatchClientMetadata().store,
+    rtlOptions: {},
+    ...options,
+  };
+  const rtlOptions = {
+    queries: {
+      ...queries,
+      queryByClassName,
+      queryByTagName,
+      getByClassName,
+      getByTagName,
+    },
+    ...internalOptions.rtlOptions,
```

Comment:
This is a bit confusing. `internalOptions` contains `rtlOptions` that can somehow be updated by `options` and then it ends up here?

Maybe define the options explicitly since `internalOptions` does not seem to be passed anywhere as an object:

```js
export const render = (ui, options = {}) => {
  const i18n = options.i18n || fakeI18n();
  const history = options.history || addQueryParamsToHistory({ history: createMemoryHistory() });
  const store = options.store || dispatchClientMetadata().store;
  const rtlOptions = {
    queries: {
      ...queries,
      queryByClassName,
      queryByTagName,
      getByClassName,
      getByTagName,
    },
    ...options.rtlOptions,
  };

  // ...
```

Reply:
Yes, I think this is better. It also means one can omit just some options and get the default values.